### ISSUE TITLE
Pin versions

### ISSUE THREAD

**Issue by skoulouzis** [QCDIS/NaaVRE#1240]
We keep getting errors in  build due to updating versions both in conda and typescript  
We may try and pin them to exact versions to have more control and which packages brake the builds 

When running:
```bash
make purge && make install-backend && make build-frontend && make install-ui && make link-ui
```

We get an error:
```bash
ValueError: 
"@jupyter_vre/vre-menu@1.0.0" is not compatible with the current JupyterLab
Conflicting Dependencies:
JupyterLab           Extension      Package
>=3.5.3 <3.6.0       >=3.6.7 <3.7.0 @jupyterlab/apputils
See the log file for details:  /tmp/jupyterlab-debug-cxbv51mo.log
make: *** [Makefile:67: install-ui] Error 1

```

**Comment by gpelouze** [comment]
I had the same problem, and fixed it temporarily on my laptop by reverting the changes of PR #1226.

It is surprising that it passed the tests… Maybe because frontend extensions are managed differently in the CI pipeline and in the local dev setup. (CI: bundled in `jupyterlab_vre-0.1.0-py3-none-any.whl`, then `jupyter serverextension enable --py jupyterlab_vre --user`. Local dev setup: built with lerna, then `cd packages/$1 && jupyter labextension [link|install] --no-build .`) 

I haven’t touched the repo, but maybe we should revert PR #1226?

**Comment by gpelouze** [comment]
Closed by #1258
